app/src/sharedTest/java/org/oppia/android/app/parser/HtmlParserTest.kt

```diff
@@ -136,6 +141,8 @@ class HtmlParserTest {
 
   @After
   fun tearDown() {
+    // Ensure lingering tasks are completed.
+    testCoroutineDispatchers.advanceUntilIdle()
```

Comment:
What exactly is the failure when this isn't included? I'm a bit surprised we need to finish anything after the test has completed.

Reply:
Ah, it was added by mistake and I left unnoticed. This has to be register and unregister `testCoroutineDispatchers`. Corrected this.